rethinkdb/datadog_checks/rethinkdb/queries_impl.py

```diff
@@ -0,0 +1,339 @@
+# (C) Datadog, Inc. 2020-present
```

Comment:
This file is adapted from the former `operations.py` file.

Contents are not exactly the same, because `operations.py` returned dicts or list of dicts, while this returns lists of tuple (to be used by the `QueryManager`). So some groupings had to be done differently.